docs/building-modules.md

```diff
@@ -30,6 +30,51 @@ Hello universe!
 
 See [example modules](../examples) for more advanced examples.
 
+## Modules
+
+Zinnia supports ES Modules (also known as
+[JavaScript Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)).
+
+For sandboxing reasons, module imports are limited to files in the root directory of the Zinnia
+module being executed.
```

Comment:
Why I choose this design:

- We can stay somewhat unopinionated about the file naming conventions in Zinnia modules.
- We are imposing only one small restriction that the module entry point must be in the root directory. 
- Developers can choose whatever file name they like - `main.js`, `index.js`, `spark.js`, etc.
- This design works both for `zinnia` CLI and `zinniad` inside Filecoin Station.